apps/subgraphs/core-dapp/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "tsconfig/server.json",
+  "include": ["**/*"],
+  "exclude": ["node_modules", ".next", ".turbo"],
+  "compilerOptions": {
+    "target": "es5",
+    "strict": true,
+    "esModuleInterop": true,
+    "outDir": "dist",
+    "resolveJsonModule": true
+  }
```

Can remove these, we already have them on the base tsconfig. It can just be

```suggestion
  "compilerOptions": {
    "target": "es5",
    "outDir": "dist"
  }
```